src/mrinufft/trajectories/utils.py

```diff
@@ -173,69 +177,220 @@ class VDSpdf(StrEnum):
+
+
+# fmt: off
+class SIEMENS:
+    """Common hardware configurations for Siemens MRI systems."""
+
+    TERRA          = Hardware(gmax=80*SI.milli, smax=200, field_strength=7)
+    TERRAX         = Hardware(gmax=200*SI.milli, smax=200, field_strength=7)
+    TERRAX_IMPULSE = Hardware(gmax=200*SI.milli, smax=900, field_strength=7)
+    PRISMA         = Hardware(gmax=40*SI.milli, smax=200, field_strength=3)
+    CIMA           = Hardware(gmax=200*SI.milli, smax=200, field_strength=3)
+    CIMAX          = Hardware(gmax=200*SI.milli, smax=200, field_strength=3)
+
+# fmt: on
+
+
+@dataclass(frozen=True)
+class Acquisition:
+    """
+    Acquisition configuration for MRI sequences.
+
+    Parameters
+    ----------
+    fov : tuple[float, float, float]
+        Field of View in meters (x, y, z).
+    img_size : tuple[int, int, int]
+        Image size in pixels (x, y, z).
+    hardware : Hardware
+        Hardware configuration for the acquisition.
+    gamma : Gammas, optional
+        Gyromagnetic ratio in Hz/T for the nucleus being imaged.
+        Defaults to `Gammas.HYDROGEN`.
+    oversampling : int, optional
+        Oversampling factor for the ADC. Defaults to 1.
+    norm_factor : float, optional
+        Normalization factor for the trajectory. Defaults to 0.5.
+
+    Attributes
+    ----------
+    default : ClassVar[Acquisition]
+        The default acquisition configuration used if none is specified.
+
+    Notes
+    -----
+    The `Acquisition` class encapsulates the parameters needed for MRI acquisition,
+    including the field of view, image size, hardware specifications, and gyromagnetic
+    ratio.
+
+    It is designed to be used in conjunction with the `Hardware` class, which defines
+    the hardware constraints such as maximum gradient amplitude and slew rate.
+    The `default` class variable holds the default acquisition configuration, which can
+    be set using the `set_default` method. This allows for easy access to a standard
+    acquisition configuration without needing to instantiate a new `Acquisition` object
+    each time.
+
+    """
+
+    default: ClassVar[Acquisition]
+
+    fov: tuple[float, float, float]  # Field of View in m
+    img_size: tuple[int, int, int]  # Image size in pixels
+    hardware: Hardware = SIEMENS.TERRA  # Hardware configuration
+    gamma: Gammas = Gammas.HYDROGEN  # Hz/T
+    oversampling: int = 1  # Oversampling factor for the ADC
+    norm_factor: float = 0.5
+
+    def set_default(self) -> Acquisition:
+        """Make the current acquisition configuration the default."""
+        Acquisition.default = self
+        return self
+
+    def __getattr__(self, name):
+        """Allow access to hardware attributes directly from Acquisition."""
+        return getattr(self.hardware, name)
+
+    @property
+    def res(self) -> tuple[float, ...]:
+        """Resolution in meters."""
+        return tuple(fov / size for fov, size in zip(self.fov, self.img_size))
+
```

Comment:
Another property:

def kmax():


This returns the kmax for this trajectory

Reply:
good idea